*** Makefile.orig	2011-03-14 18:41:39.000000000 +0100
--- Makefile	2011-03-14 18:42:48.000000000 +0100
***************
*** 1,8 ****
  install:
! 	cp  src/screenfo /usr/local/bin
! 	mkdir -p /usr/share/screenfo
! 	cp -r ascii/ /usr/share/screenfo
! 	cp screenfo.conf /usr/share/screenfo/screenfo.conf.example
  
  uninstall:
  	rm -fv /usr/local/bin/screenfo
--- 1,8 ----
  install:
! 	cp  src/screenfo ${DESTDIR}/usr/local/bin
! 	mkdir -p ${DESTDIR}/usr/share/screenfo
! 	cp -r ascii/ ${DESTDIR}/usr/share/screenfo
! 	cp screenfo.conf ${DESTDIR}/usr/share/screenfo/screenfo.conf.example
  
  uninstall:
  	rm -fv /usr/local/bin/screenfo
